src/domains/printable.ml

```diff
@@ -120,7 +120,6 @@ struct
   let name () = "HConsed "^Base.name ()
   let hash x = x.BatHashcons.hcode
   let tag x = x.BatHashcons.tag
-  let equal x y = x.BatHashcons.tag = y.BatHashcons.tag
```

Comment:
why did we remove the equal here?

Reply:
There is a another copy of this defintion of `equal` lower down (line 142/143). So this first definition is just redundant.